api/commands/reminders/set.js

```diff
@@ -13,6 +13,35 @@ const queue = kue.createQueue({
   }
 });
 
+const commands = {

```

Comment:
@JustinHallquist Why is this called `commands` when we're really only talking about a single cmd?

Reply:
oversight. will update